Ghidra/Features/Python/ghidra_scripts/python_basics.py

```diff
@@ -46,7 +46,7 @@
 if type(my_dictionary) == type(dict):
     print "my_dictionary is a dictionary!"
 
-if my_null is None and 2 + 2 == 4:
+if my_null is None:
     print "my_null is None and 2 + 2 == 4!"
```

Comment:
Clearly this was done on purpose as they wouldn't have added 2 + 2 == 4 to the print statement as well.